### PR TITLE
Closes #236 (update resolvers and GQL schema)

### LINKED ISSUE #236: feature/update resolvers and GQL schema
After updating DB schema, we need to update code in GraphQL resolvers and GraphQL schema

### PR DESCRIPTION
**Pull request by jpmonettas**
Includes:
- graphql schema update
- resolvers update
- syncer update
- smart contracts changes for how we handle arbiters and candidates
- db changes

### CODE REVIEW

File: src/ethlance/server/graphql/resolvers.cljs [hunk 1]
```diff
@@ -442,5 +512,12 @@
                     :Arbiter {:arbiter_feedback arbiter->feedback-resolver}
                     :Feedback {:feedback_toUserType feedback->to-user-type-resolver
                                :feedback_fromUserType feedback->from-user-type-resolver}
-                    :Mutation {:signIn sign-in-mutation}}
-  )
+                    :Mutation {:signIn sign-in-mutation
```
Comment (madvas) [inline src/ethlance/server/graphql/resolvers.cljs:515]: We're missing here `createInvoice` `payInvoice` `cancelInvoice` `createJobProposal`

File: contracts/EthlanceJobs.sol [hunk 2]
```diff
@@ -776,6 +763,6 @@ contract EthlanceJobs {
   event JobApproversUpdated(uint _jobId, address _changer, address[] _approvers);
   event JobDataChanged(uint _jobId, address _changer, string _ipfsHash);
 
-  event CandidateAccepted(uint jobId, address candidate);
-  event CandidateApplied(uint jobId, address candidate);
+  event CandidateAccepted(uint _jobId, address _candidate);
+  event CandidateApplied(uint _jobId, address _candidate, uint _rate, string _proposalIpfsHash);
```
Comment (madvas) [inline contracts/EthlanceJobs.sol:767]: This event should be deleted

File: src/ethlance/server/graphql/resolvers.cljs (hunk 1 above)
Comment (jpmonettas) [reply src/ethlance/server/graphql/resolvers.cljs:528]: Addressed

File: contracts/EthlanceJobs.sol (hunk 2 above)
Comment (jpmonettas) [reply contracts/EthlanceJobs.sol]: Done

**Pull request merged**